modules/ROOT/pages/insight.adoc

```diff
@@ -82,33 +74,30 @@ You can configure business events in your Mule applications by embedding custom
 
 [NOTE]
 ====
-Note that when deploying your application on Runtime Manager, you must make sure that in the *Insight* tab you select either *Metadata* or *Metadata and Replay*
-
-image::chinsighttab.png[insight tab]
+When deploying your application on Runtime Manager, you must make sure that in the *Insight* tab you select either *Metadata* or *Metadata and Replay*.
 ====
 
-
 == Searching and Filtering Transactions and Events
 
-The *Insight Dashboard* displays information at three tracking levels: Transactions, Events, and Metadata.
+The *Insight Dashboard* displays information at three tracking levels: 
 
-* Transactions are logical groupings of related events, and often correspond to a business view of the system.
-* Events track low-level details of a transaction. Events map to message processors and endpoints and reveal information about exceptions and any custom business events you may have configured.
-* Metadata tracks customized key/value pairs that you have specified as part of custom events in your application. The information that Insight tracks at this level presents information about high-level business events.
+* Transactions: Logical groupings of related events that often correspond to a business view of the system.
+* Events: Low-level details of a transaction. Events map to message processors and endpoints and reveal information about exceptions and any custom business events you may have configured.
+* Metadata: Tracks customized key/value pairs that you specify as part of custom events in your application. The information that Insight tracks at this level presents information about high-level business events.
 
 The following image illustrates where to find transactions, events, and metadata on the Insight dashboard.
 
 image::chindash2.png[CHInDash2]
 
 == Customizing the Transaction ID
 
-For Transaction-level events, Insight may display long numeric values in the ID column. Rather than use random number identifiers, you may wish to customize the transaction IDs so that they display more intuitive values which are relevant to your business requirements.
+For transaction-level events, Insight may display long numeric values in the ID column. Rather than use random number identifiers, you can customize the transaction IDs so that they display more intuitive values that are relevant to your business requirements.
```

Comment:
delete "you can" (also, are hte long numeric values the random numbers? If so, how about "Rather than use these random number identifiers"?